### PR TITLE
Added documentation for `vault operator usage`.

### PR DESCRIPTION
**Pull request by mgritter**

### CODE REVIEW

File: website/pages/docs/commands/operator/usage.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# operator usage
+
+The `operator usage` command allows an administrator to retrieve a
+[client count](https://www.vaultproject.io/docs/concepts/client-count) report
+for the default reporting period, or for a specific time range of months.
+
+The command output will list distinct entities, non-entity tokens, and total
+active clients for each namespace, and the total for the entire cluster.  It may
+indicate that no data is available for the requested time range, which may be
+because the collection of client count is disabled, or the time range is too far
```
Comment (swayne275) [inline website/pages/docs/commands/operator/usage.mdx:18]:
```suggestion
because the client count reporting is disabled, the time range is too far
```

**Review by swayne275 — APPROVED** [review]
Looks good! I think the suggestion I made would make it a little easier to grok

File: website/pages/docs/commands/operator/usage.mdx (hunk 1 above)
Comment (mgritter) [reply website/pages/docs/commands/operator/usage.mdx:18]: Sure. I was trying to make a distinction between collection (which might be disabled) and reporting (which never is.)

Comment (swayne275) [reply website/pages/docs/commands/operator/usage.mdx]: Oh, that makes sense. Maybe `client count <some word instead of reporting>` then?

**Pull request merged**